presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -191,6 +176,63 @@ Property Name                                      Description
                                                    :ref:`s3selectpushdown`.
 ================================================== ============================================================ ============
 
+Hive Thrift Metastore Configuration Properties
+----------------------------------------------
+
+================================================== ============================================================ ============
+Property Name                                      Description                                                  Default
+================================================== ============================================================ ============
+``hive.metastore.uri``                             The URI(s) of the Hive metastore to connect to using the
+                                                   Thrift protocol. If multiple URIs are provided, the first
+                                                   URI is used by default and the rest of the URIs are
+                                                   fallback metastores. This property is required.
+                                                   Example: ``thrift://192.0.2.3:9083`` or
+                                                   ``thrift://192.0.2.3:9083,thrift://192.0.2.4:9083``
+
+``hive.metastore.username``                        The username Presto will use to access the Hive metastore.
+
+``hive.metastore.authentication.type``             Hive metastore authentication type.                          ``NONE``
```

Comment:
Since there's only one property with a default, let's remove the default column and write the default as part of the description, like we do for the S3 properties